src/GUARDRAIL 6 PROTECTION OF DATA-AT-REST/Audit/Check-ProtectionDataAtRest.psm1

```diff
@@ -35,35 +76,197 @@ function Check-StatusDataAtRest {
         $ReportTime
     )   
     [PSCustomObject] $tempObjectList = New-Object System.Collections.ArrayList
+
     foreach ($obj in $objList)
     {
+        Write-Verbose "Checking $objType : $($obj.Name)"
+        Write-Verbose "PBMM policy PolicyID is $PolicyID"
+
+        # Find scope
         if ($objType -eq "subscription"){
             $tempId="/subscriptions/$($obj.Id)"
         }
         else {
-            $tempId=$obj.Id
+            $tempId=$obj.Id                              
         }
+        Write-Host "Scope is $tempId"
```

Comment:
Guess this was for debugging only. @dutt0

Reply:
Yes. its for only debugging in local execution